pywbem_mock/_wbemconnection_mock.py

```diff
@@ -1488,9 +1508,19 @@ def _fake_enumerateclassnames(self, namespace, **params):
 
             CIMError: CIM_ERR_INVALID_NAMESPACE if invalid namespace,
             CIMError: CIM_ERR_NOT_FOUND if Classname not found
+            CIMError: CIM_ERR_INVALID_CLASS if class that is basis for tes
+                      does not exist
```

Comment:
See previous comments.

Reply:
See above.  I think this is fixed unless we want to put try block around the GetClass loop and output an exception about the repository integrity in place of the CIM_ERR_NOT_FOUND. I did not want to ignore that error if it ever occured but it could only occur if one function found the class and right after another did not find the same class in the repo.  

After all, this is a test tool.